app/models/addon-external-account.ts

```diff
@@ -0,0 +1,18 @@
+import { AsyncHasMany, attr, hasMany } from '@ember-data/model';
+
+import OsfModel from './osf-model';
+import OsfUserModel from './osf-user';
+
+export default class AddonExternalAccountModel extends OsfModel {
```

Comment:
I don't see this in the API contract, other than it saying that it won't be exposed via the API. Did you talk with Jon about this?

Reply:
Update from our models implemenation mobbing: `external_accounts` will not be surfaced in the API